### ISSUE TITLE
Problem with hooks when building distribution

### ISSUE THREAD

**Issue by sphuber** [DropD/reentry#17]
I was trying to build the `aiida-metapkg` to upload to PyPi in a virtual environment that had `reentry==1.0.2` installed and so when I ran the following:

```
python setup.py sdist bdist_wheel
```

I received the following exception.

```
Traceback (most recent call last):
  File "setup.py", line 13, in <module>
    **kwargs
  File "/usr/lib/python2.7/distutils/core.py", line 111, in setup
    _setup_distribution = dist = klass(attrs)
  File "/home/sphuber/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/setuptools/dist.py", line 338, in __init__
    _Distribution.__init__(self, attrs)
  File "/usr/lib/python2.7/distutils/dist.py", line 287, in __init__
    self.finalize_options()
  File "/home/sphuber/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/setuptools/dist.py", line 471, in finalize_options
    ep.load()(self, ep.name, value)
  File "/home/sphuber/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/reentry/hooks.py", line 29, in register_dist
    jb.write_st_dist(dist)
  File "/home/sphuber/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/reentry/jsonbackend.py", line 68, in write_st_dist
    epmap = dist.entry_points.copy()
AttributeError: 'NoneType' object has no attribute 'copy'
```

Uninstalling `reentry` from the virtual environment allowed me to build the package again.

**Comment by DropD** [comment]
This will require some investigation

**Comment by ltalirz** [comment]
Note: If you put
```
    "entry_points": {
    },
```
into the `setup.json` of the aiida meta package, the problem goes away, i.e. it is really a problem of reentry.

**Comment by DropD** [comment]
fixed in v1.1.1
